android/.settings/org.eclipse.buildship.core.prefs

```diff
@@ -0,0 +1,9 @@
+arguments=
+auto.sync=false
+gradle.user.home=
```

Comment:
Sorry, I meant the whole file, it should be ignored by the gitignore

Reply:
I have just remove this file and .settings already in gitignore